README.md

```diff
@@ -40,7 +40,8 @@ probe('http://example.com/image.jpg', function (err, result) {
       type: 'jpg',
       mime: 'image/jpeg',
       wUnits: 'px',
-      hUnits: 'px'
+      hUnits: 'px',
+      responseUrl: 'http://example.com/image.jpg'
```

Comment:
Sounds great – the hope with to `responseUrl` was to convey that it wasn't necessarily the original url, but I think either works fine